### PR TITLE
feat(frontend): Improved footer design ESP #471

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- resolves https://github.com/eoscostarica/guide.eoscostarica.io/issues/471

Note: react-icons was installed: npm install react-icons --save or yarn install react-icons

### Steps to test
1. Navigate to the bottom of [guias.eosio](https://guias.eoscostarica.io/) and see page footer

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by fvives99** [comment]
image

**Review by JAMares — CHANGES_REQUESTED** [review]
See the LGTM comments about unused and use yarn build and npm run serve to see the final result

**Review by fvives99 — COMMENTED** [review]
fixed

File: src/theme/Footer/index.js [hunk 1]
```diff
@@ -4,95 +4,162 @@
...
+   const { copyright, logo = {} } = footer || {};
+   const [logoUrl, setLogoUrl] = useState(useBaseUrl(logo.src));
+   const { colorMode } = useColorMode();
+  
+   if (!footer) {
+     return null;
+   }
+ 
+   useEffect(() => {
+     console.log(isDarkTheme)
```
Comment (AngeloCG97) [inline src/theme/Footer/index.js:39]: Please remove this comment

Comment (fvives99) [reply src/theme/Footer/index.js]: removed.

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Comment by fvives99** [comment]
image
@AngeloCG97 mobile footer

**Pull request merged**